packages/react/src/components/Coachmark/Coachmark.test.tsx

```diff
@@ -30,66 +30,63 @@ describe('Coachmark', () => {
   it('renders Coachmark (correctly)', () => {
     ReactDOM.createPortal = jest.fn(element => element);
 
-    safeCreate(
+    const { container } = render(
       <Coachmark className={'test-className'} target="test-target">
         This is a test
       </Coachmark>,
-      component => {
-        const tree = component!.toJSON();
-        expect(tree).toMatchSnapshot();
-      },
     );
+    expect(container).toMatchSnapshot();
   });
 
   it('renders Coachmark (isCollapsed)', () => {
     ReactDOM.createPortal = jest.fn(element => element);
 
-    safeCreate(
+    const { container } = render(
       <Coachmark isCollapsed={false} className={'test-className'} target="test-target">
         This is a test
       </Coachmark>,
-      component => {
-        const tree = component!.toJSON();
-        expect(tree).toMatchSnapshot();
-      },
     );
+    expect(container).toMatchSnapshot();
   });
 
   it('renders Coachmark (color properties)', () => {
     ReactDOM.createPortal = jest.fn(element => element);
 
-    safeCreate(<Coachmark beaconColorOne="green" beaconColorTwo="blue" color="red" target="test" />, component => {
-      const tree = component!.toJSON();
-      expect(tree).toMatchSnapshot();
-    });
+    const { container } = render(<Coachmark beaconColorOne="green" beaconColorTwo="blue" color="red" target="test" />);
+    expect(container).toMatchSnapshot();
   });
 
   // Accessibility Tests:
   it('correctly applies (ariaAlertText)', () => {
-    safeMount(<Coachmark ariaAlertText="aria alert " target="test-target" />, component => {
-      expect(component.find('[role="alert"]').getDOMNode()).toBeDefined();
-    });
+    const { getByRole } = render(<Coachmark ariaAlertText="aria alert " target="test-target" />);
+    expect(getByRole('alert')).toBeTruthy();
   });
 
   it('correctly applies (ariaLabelBy)', () => {
-    safeMount(
-      <Coachmark ariaLabelledBy="aria label" ariaLabelledByText="aria text" target="test-target" />,
-      component => {
-        expect(component.find('[role="dialog"]').getDOMNode().getAttribute('aria-labelledby')).toBe('aria label');
-        expect(component.find('[id="aria label"]').text()).toBe('aria text');
-      },
+    const { getByRole } = render(
+      <Coachmark
+        ariaLabelledBy="aria label"
+        ariaLabelledByText="aria text"
+        target={document as ICoachmarkProps['target']}
+      />,
     );
+
+    const coachmark = getByRole('dialog', { hidden: true });
```

Comment:
Needed `hidden:true` option since `dialog` role is inaccessible here because the element containing that role has a parent wrapper with `role=presentation` when the `Coachmark` is not collapsed.

Reply:
Interesting, I wonder if that's an actual a11y problem? Regardless would be good to note that in a comment in the file.